proxy/stats_get_prediction.go

```diff
@@ -92,6 +94,16 @@ func (s *StatsGetPrediction) HandleGetStats(w http.ResponseWriter, r *http.Reque
 		r.Body.Close()                                        //  must close
 		r.Body = ioutil.NopCloser(bytes.NewBuffer(bodyBytes)) // Reset Body as the request gets reused by catchall if this has an error.
 		req := string(bodyBytes)
+		if strings.HasSuffix(r.RequestURI, "catalog/get_replay") {
+			regex := regexp.MustCompile(`940100059aff00.*$`)
+			for _, data := range []string{"940100059aff00636390ffff000001\x00", "940100059aff00636390ffff010001\x00", "940100059aff00636390ffff020001\x00"} {
```

Comment:
1. Is the `0x00` necessary at the end? Strings are implied to be null terminated.
2. I assume these are just 3 hard-coded random replays? Can we just hard-code the response instead?

Reply:
1. If I remove the 0x00 it doesn't work anymore. Getting cache misses instead.
2. The body of the get_replay requests is just a search filter (see [this comment](https://github.com/optix2000/totsugeki/issues/35#issuecomment-922516535)). The filters I'm using here just says celestial floor random characters. Doing it like this already puts the response time below 1ms. Returning hard-coded replays instead might break in the future if the replays are from an older game version.